deterministic-bloom/tests/integration_test.rs

```diff
@@ -1,4 +1,16 @@
+use deterministic_bloom::{test_utils::Rvg, BloomFilter};
+
 #[test]
-fn test_add() {
-    assert_eq!(deterministic_bloom::add(3, 2), 5);
+fn test_contains() {
```

Comment:
does this cover all the tests (unit + integration) from the original port? I can also double check :).